cmd/argo/commands/common.go

```diff
@@ -21,6 +21,7 @@ var (
 	restConfig       *rest.Config
 	clientConfig     clientcmd.ClientConfig
 	clientset        *kubernetes.Clientset
+	wfClientset      *wfclientset.Clientset
```

Comment:
Why now use `wfClient`

Reply:
I couldn't access the rest client from wfClient in order to make a POST request.